src/libstd/prelude/mod.rs

```diff
@@ -11,8 +11,8 @@
 //! The Rust Prelude
 //!
 //! Because `std` is required by most serious Rust software, it is
-//! imported at the topmost level of every crate by default, as if the
-//! first line of each crate was
+//! imported at the topmost level of every crate by default, as if
+//! each crate contains the following:
```

Comment:
So to me, 'contains' sounds like it's the entire contents, not starting with. I would maybe say

> Because `std` is required by most serious Rust software, it is imported at the topmost level of each crate by default. If this were explicit, it might look like this:

```ignore
extern crate std;

// your code here


like that. Whatcha think?